urbanopt/geojson-modelica-translator
# refactor!: use gmt class to generalize cli DES generation

#### Any background context you want to provide?
GMT CLI currently only supports DES with timeseries loads.

#### What does this PR accomplish?
Allows CLI to generate packages with any load types (e.g. timeseries, spawn, teaser)

#### How should this be manually tested?
Create a geojson and sys param with one or more loads (at least one non-timeseries load), and run
```
poetry run uo_des create-model <sys param path> <geojson path> <output dir>
```
Verify the package looks correct (most importantly the correct load types were used). Try and simulate in Dymola.

#### What are the relevant tickets?
#370
